test/OpenTelemetry.Tests.Stress.Metrics/Program.cs

```diff
@@ -24,18 +24,16 @@ namespace OpenTelemetry.Tests.Stress;
 
 public partial class Program
 {
-    private const int ArraySize = 10;
-
-    // Note: Uncomment the below line if you want to run Histogram stress test
-    // private const int MaxHistogramMeasurement = 1000;
+    private const int ArraySize = 49;
```

Comment:
I don't think we should update the Stress Tests. Let's keep it simple here. If anyone wants to test a specific scenario, they can always update it here, but we don't have to check in a specific scenario to the main branch.

Reply:
Ya that's fair I could submit a separate PR that makes it simpler to stress test different instruments which was all I really wanted to achieve with the changes I made.